gqlalchemy/memgraph.py

```diff
@@ -489,6 +495,12 @@ def save_relationship(self, relationship: Relationship) -> Optional[Relationship
         result = self._save_relationship_properties_on_disk(relationship, result)
         return result
 
+    def save_relationships(self, relationships: List[Relationship]) -> None:
+        """Saves a list of relationships to Memgraph."""
+        for i in range(len(relationships)):
+            saved_relationship = self.save_relationship(relationships[i])
```

Comment:
Is this line necessary?

Reply:
The one above or below?

Reply:
It was the one below. I removed the unnecessary variable assignment.